server/repository/articleRepository.js

```diff
@@ -110,6 +112,64 @@ class ArticleRepository {
     }
     return article;
   }
+
+  /**
+   * Adds an article to  a user's reading stats
+   * @param {object} user Reader
+   * @param {object} article Read article
+   * @returns {object} New or updated reading stat
+   */
+  static async addReadingStat(user, article) {
+    const hasRead = await user.hasReadArticle(article);
+
+    if (!hasRead) {
+      const newRead = await user.addReadArticle(article);
+      return newRead[0][0].dataValues;
+    }
+
+    const justUpdated = await ReadingStat.update(
```

Comment:
@PriscillaSam I think it will be better if we create a new row instead of updating the old one. That way we can be able to aggregate the number of times a user has read an article.

Reply:
Or we could have a column that records the number of time a user has read the article.

Reply:
@TheoOkafor I feel that would not be necessary as we would be repeating rows in the database.